images/linux/scripts/base/apt-mock.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash -e
 
+set -x
```

Comment:
Why do we want to invoke this command?
It prints all script to logs, do we really want it?

Reply:
The same for `which $tool || continue`. Let's get rid of additional output